ios/Classes/SwiftFfFlutterClientSdkPlugin.swift

```diff
@@ -200,8 +200,18 @@ public class SwiftFfFlutterClientSdkPlugin: NSObject, FlutterPlugin {
                 }
 				}
 
-			case .destroy:
-				CfClient.sharedInstance.destroy()
+            case .destroy:
+                CfClient.sharedInstance.destroy { result in
+                    DispatchQueue.main.async {
+                        switch result {
+                        case .success:
+                            print("SDK destroyed successfully.")
+                        case .failure(let reason):
+                            print("Failed to destroy SDK: \(reason)")
+                        }
+                        result(true)
```

Comment:
Always return the result Future as `true`.  The log is sufficient. It would require a change to the core Fluter `destroy` method to handle anything else, and an update to all three plugins.